spec/cli_spec.rb

```diff
@@ -27,7 +27,7 @@
       cmd = %w[bundle exec rails-response-dumper --exclude-timestamp]
       stdout, stderr, status = Open3.capture3(*cmd, chdir: APP_DIR)
       expect(stderr).to eq('')
-      expect(stdout).to eq(".....\n")
+      expect(stdout).to match(/.+\n/)
```

Comment:
Surround this regex with `\A` and `\z` so we're confident that we're asserting everything. (for each `match`)